R/lazy-join-query.R

```diff
@@ -7,6 +7,9 @@ lazy_join_query <- function(x,
                             by,
                             suffix = c(".x", ".y"),
                             na_matches = c("never", "na"),
+                            group_vars = NULL,
```

Comment:
`add_group_by()` adds the field `group_vars` to any `lazy_query` object. Similarly for `order_vars` and `frame`. So it makes sense to be able to explicitly set them when creating these objects.